### ISSUE TITLE
Architecture and Tech Stack

### ISSUE THREAD

**Issue by AmirNaghibi** [AmirNaghibi/BookBits#6]
Discuss in high-level:

- what is the main goal of the project
- overview of the architecture
- what technology stack we will be using.

**Comment by engichang1467** [comment]
Let's first list out the requirements of BookBits
Then, we can use draw.io to sketch out the architecture of our application

**Comment by AmirNaghibi** [comment]
Agree. I like your idea of versioning the software. We can plan out a basic MVP. Let's keep things simple. 

**Basically, the website is for book summaries. We have a list of books, for each book, there is a list of information in the point format about the book**

## 12 Rules For Life – by Jordan Peterson:

- Stand up straight with your shoulders back
- Treat yourself like someone you are responsible for helping
- Make friends with people who want the best for you
- Compare yourself to who you were yesterday, not to who someone else is today
- Do not let your children do anything that makes you dislike them
- Set your house in perfect order before you criticize the world
- Pursue what is meaningful (not what is expedient)
- Tell the truth – or, at least, don’t lie
- Assume that the person you are listening to might know something you don’t
- Be precise in your speech
- Do not bother children when they are skateboarding
- Pet a cat when you encounter one on the street

**Comment by AmirNaghibi** [comment]
There should also be a form for others to add their notes to an existing book or create a new book that does not exist in our database and add their notes
